Lesson 5 - CSS/code/exercises/exercise 1/theme.css

```diff
@@ -1,6 +1,66 @@
 .row {
     display: block;
+    font-family: 'Times New Roman', Times, serif;
+    margin: auto;
+    width: 1000px;
 }
-
-
-
+.column
+{
+    float: left;
+    width: 500px;
+}
+h1[title^="t1"]

```

Comment:
This semantic selectors from *CSS3* are little bit better :+1:  that plain `id` but you are able to write your CSS with any changes in HTML structure. Try doing this with `display: inline-block;`